src/wwwroot/controllers/dKimConfigCtrl.js

```diff
@@ -44,5 +46,15 @@
         vm.canoncalTrackingDomain = response.data.canonical_tracking_domain;
       });
   }
+
+  vm.sendResubscribe = function(form) {
+    vm.submitted = true;
+
+    if (!form.$valid) {
+      return;
+    }
+
+    settings.resubscribeEmailAddress(vm.email, vm.reason, vm.domain, false);
```

Comment:
I think that the 4th parameter is not necessary (at least by the moment)

```suggestion
    settings.resubscribeEmailAddress(vm.email, vm.reason, vm.domain);
```